BaseInterfaces/test/runtests.jl

```diff
@@ -2,55 +2,16 @@ using BaseInterfaces
 using Interfaces
 using Test
 
-@implements SetInterface{(:empty,:emptymutable,:hasfastin,:intersect,:union,:sizehint!)} Test.GenericSet
-@implements DictInterface Test.GenericDict
+@implements SetInterface{(:empty,:emptymutable,:hasfastin,:intersect,:union,:sizehint!)} Test.GenericSet [Test.GenericSet(Set((1, 2)))]
```

Comment:
Why do you need those in addition to `Interfaces.test(BaseInterfaces)`?

Reply:
Oh yeah they will be in `Main`.